app/jinja_filters.py

```diff
@@ -26,21 +26,26 @@ def nl2br(context, value):
 
 
 @blueprint.app_template_filter()
-def pretty_date(value):
+def format_date(value):
     return value.strftime('%-d %B %Y')
 
 
-def pretty_short_date(value):
-    return datetime.strptime(value, "%d/%m/%Y").strftime('%d %b %Y')
+def format_str_as_short_date(value):
```

Comment:
Making it explicit due to the fact we have other Jinja filters formating dates that aren't strings